### ISSUE TITLE
A11y_30days_Welcome to 30days of learning_Use of Color: Underline is not provided for the links present under “30Days of Learning” heading.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#172]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#WCAG1.4.1;#DesktopWeb;#Win11;ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/ 
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/
2.	“30Days of Learning” page will open.
3.	Tab to links present under “30Days of Learning” heading.
4.	Observe that the underline is not provided for the links.
### Actual:
Underline is not provided for the links present under “#30Days of Learning” heading.
Note: This issue is observed with all the links present in the page.
### Expected:
Underlines should be provided for the links present under “#30Days of Learning” heading.
### User Impact:
Low vision users will not be able to differentiate between normal text and link.

### Attachment:
<img width="960" alt="1_A11y_  WCP-CELAP3 30days_Welcome to 30days of learning_Use of Color Underline is not provided for the links present under 30Days of Learning heading" src="https://user-images.githubusercontent.com/85863895/209324391-997e8e41-385c-46a6-9137-12f8a9ccc10f.png">

**Comment by nitya** [comment]
@Ashutoshsahoo164 can you verify if issue is resolved?

**Comment by Ashutoshsahoo164** [comment]
Hi @nitya The bug is still reproducing . I have verified in this URL : https://microsoft.github.io/30daysof/

![Screenshot (16)](https://user-images.githubusercontent.com/85863895/220595228-6c1280e5-ff7e-4156-9f40-98eb6ec4cbe7.png)

**Comment by Ashutoshsahoo164** [comment]
As the Site is now not be published , closing this bug.
